From: Egbert Eich <eich@suse.de>
Date: Thu Aug 28 19:55:39 2014 +0200
Subject: drm/panic: Set priority of the drm panic code to the lowest value possible
Patch-mainline: to be upstreamed

References: bnc#892114
Signed-off-by: Egbert Eich <eich@suse.com>

This makes sure it will be placed at the very end of the call chain.
This way any notifier with a higher priority will have the chance to
run in case the mode setting code hangs the system - which may happen
as it was not written to run in a panic context.

Signed-off-by: Egbert Eich <eich@suse.de>
---
 drivers/gpu/drm/drm_fb_helper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/drm_fb_helper.c b/drivers/gpu/drm/drm_fb_helper.c
index 92d077e..1b207ee 100644
--- a/drivers/gpu/drm/drm_fb_helper.c
+++ b/drivers/gpu/drm/drm_fb_helper.c
@@ -356,6 +356,7 @@ static int drm_fb_helper_panic(struct notifier_block *n, unsigned long ununsed,
 
 static struct notifier_block paniced = {
 	.notifier_call = drm_fb_helper_panic,
+	.priority = INT_MIN,
 };
 
 static bool drm_fb_helper_is_bound(struct drm_fb_helper *fb_helper)
